test/e2e/e2e_setup_test.go

```diff
@@ -185,23 +144,126 @@ func Setup(specName string, f func(testSpecificSettings func() testSettings), op
 			Variables:            testSpecificVariables,
 		})
 
-		if testMode == SupervisorTestMode {
-			runtimeExtensionProviders = append(runtimeExtensionProviders, "vm-operator", "net-operator")
-		}
+		postNamespaceCreatedFunc = func(managementClusterProxy framework.ClusterProxy, workloadClusterNamespace string) {
```

Comment:
Please add a comment to explain why all of this has to happen after namespace creation

Reply:
Feels a bit off to document it but okay for me. It is not documented elsewhere for other things "why something needs to happen here".